### ISSUE TITLE
Error when enabling Meteor package

### ISSUE THREAD

**Issue by ebroder** [zodern/meteor-types#2]
I tried adding `zodern:types` to a project of mine and it seems to be causing `meteor lint` to crash more or less immediately with a somewhat inscrutable error:

```
/root/.meteor/packages/meteor-tool/.2.7.3.1mjjp3y.wu0o++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:218
      throw error;
      ^

Error: conflicts in compiler?
    at /tools/isobuild/package-source.js:68:[15](https://github.com/deathandmayhem/jolly-roger/runs/7797579066?check_suite_focus=true#step:7:16)
    at memoize (/root/.meteor/packages/meteor-tool/.2.7.3.1mjjp3y.wu0o++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/underscore/underscore-node-f-pre.js:1061:55)
    at /tools/isobuild/package-source.js:97:24
    at Array.sort (<anonymous>)
    at SourceArch.getFiles (/tools/isobuild/package-source.js:969:58)
    at lintUnibuild (/tools/isobuild/compiler.js:321:32)
    at /tools/isobuild/compiler.js:228:30
    at Function.each (/root/.meteor/packages/meteor-tool/.2.7.3.1mjjp3y.wu0o++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/underscore/underscore-node-f-pre.js:13[16](https://github.com/deathandmayhem/jolly-roger/runs/7797579066?check_suite_focus=true#step:7:17):7)
    at Object.lint (/tools/isobuild/compiler.js:221:5)
    at lintBundle (/tools/isobuild/bundler.js:3488:54)
    at /tools/isobuild/bundler.js:3322:25
    at Object.capture (/tools/utils/buildmessage.js:283:5)
    at bundle (/tools/isobuild/bundler.js:3243:31)
    at /tools/isobuild/bundler.js:3[18](https://github.com/deathandmayhem/jolly-roger/runs/7797579066?check_suite_focus=true#step:7:19)6:32
    at Slot.withValue (/root/.meteor/packages/meteor-tool/.2.7.3.1mjjp3y.wu0o++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/@wry/context/lib/context.esm.js:69:29)
    at Object.withCache (/root/.meteor/packages/meteor-tool/.2.7.3.1mjjp3y.wu0o++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/tools/fs/tools/fs/files.ts:1543:39)
    at Object.bundle (/tools/isobuild/bundler.js:3186:16)
    at Command.func (/tools/cli/commands.js:1310:[26](https://github.com/deathandmayhem/jolly-roger/runs/7797579066?check_suite_focus=true#step:7:27))
    at /tools/cli/main.js:15[34](https://github.com/deathandmayhem/jolly-roger/runs/7797579066?check_suite_focus=true#step:7:35):15
```

(If helpful, you can see the PR here: deathandmayhem/jolly-roger#993)

**Comment by zodern** [comment]
Thanks for letting me know. This is a bug in Meteor, and while working on this I found another bug with Meteor linters. Both are fixed in https://github.com/meteor/meteor/pull/12120.

**Comment by zodern** [comment]
I released version zodern:types@1.0.9 which tries to avoid this bug.
